.github/workflows/exhaustive.yml

```diff
@@ -61,6 +61,6 @@ jobs:
         run: >
           mvn -s .github/workflows/mvn-settings.xml -B test --file tools/pom.xml
           -pl cldr-code -Dorg.unicode.cldr.unittest.testArgs='-e10 -n'
-          -DCLDR_GITHUB_ANNOTATIONS=true
+          -DCLDR_GITHUB_ANNOTATIONS=true -DCLDR_TEST_ENABLE_NET=true
```

Comment:
link check did not activate during exhaustive test due to `-pl cldr-code` … will try something else